webserver/templates/user/export.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+{% block title %}Export for "{{ user.musicbrainz_id }}" - ListenBrainz{% endblock %}
+
+{% block content %}
+  {% if current_user.is_authenticated() and current_user == user %}
+    <h3>Export from ListenBrainz</h3>
+    <p>
+      <strong> Notes about the ListenBrainz export process : </strong>
+    </p>
+    <p>
+        Now you can export your listens to a file. 
+        Just click on the download and let ListenBrainz do the hard work of figuring out what to do.
```

Comment:
Please update lines 11 & 12 to say:

Export your listen history in JSON format and download it. Click Download to proceed.